src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -49,7 +49,7 @@ internal class GrpcWorkerChannel : IRpcWorkerChannel, IDisposable
         private readonly IOptions<WorkerConcurrencyOptions> _workerConcurrencyOptions;
         private readonly WaitCallback _processInbound;
         private readonly object _syncLock = new object();
-        private readonly Dictionary<MsgType, Queue<PendingItem>> _pendingActions = new ();
+        private readonly Dictionary<MsgType, Queue<PendingItem>> _pendingActions = new Dictionary<MsgType, Queue<PendingItem>>();
```

Comment:
What settings do I need to build this locally? Something that I have configured automatically removes that space, then it fails to build, then I re-add it, etc. But it only happens if I ever open this file. I literally cannot run locally with this like this, and this was the easiest solution.

I suspect others would hit this as well... if there's some editerconfig I can use to make this work, I'd gladly do it.

I tried moving to a newer StyleCop, but that opened up a whole new can of worms with dozens of newer violations that it wanted me to address.

Reply:
That space _should_ be removed. The style is typically `new()` and not `new ()`. What error are you getting for the failed build? I see we don't have a `global.json` to lock the dotnet SDK in place, so this could be a case of you having an older dotnet SDK?

Reply:
This was fixed in #8975. Have rebased on that change.